celery_serverless/handler.py

```diff
@@ -60,32 +63,52 @@ def worker(event, context, intercom_url=None):
 
 @handler_wrapper
 def watchdog(event, context):
-    lock_name = os.environ.get('CELERY_SERVERLESS_LOCK_NAME', 'celery_serverless:watchdog')
-    lock_url = os.environ.get('CELERY_SERVERLESS_LOCK_URL')
-    assert lock_url, 'The CELERY_SERVERLESS_LOCK_URL envvar should be set. Even to "disabled" to disable it.'
-
     queue_url = os.environ.get('CELERY_SERVERLESS_QUEUE_URL')
     assert queue_url, 'The CELERY_SERVERLESS_QUEUE_URL envvar should be set. Even to "disabled" to disable it.'
```

Comment:
Letting it break with `queue_url = os.environ['CELERY_SERVERLESS_QUEUE_URL']` and raising `KeyError` might make the code a bit simples.

Reply:
But then it gives less clue about how to solve the problem. "Is it a code error or my mistake?".

To change the message of a KeyError it will get as long as it is now. Is it worth?

Reply:
I would be ok without a specific message. Seeing `os.environ['CELERY_SERVERLESS_QUEUE_URL']` on the error is enough for me to know that I am missing that envvar. But whether to change or not is up to you.